app/src/main/java/de/perflyst/untis/activity/ActivityLoginDataInput.java

```diff
@@ -9,20 +9,11 @@
 import android.util.Log;
 import android.util.Patterns;
 import android.view.View;
-import android.widget.ArrayAdapter;
-import android.widget.AutoCompleteTextView;
-import android.widget.Button;
-import android.widget.EditText;
-import android.widget.ImageView;
-import android.widget.ProgressBar;
-import android.widget.RelativeLayout;
-import android.widget.TextView;
-
+import android.widget.*;
```

Comment:
Why should we use widget.* in this case instead of specific import's?

Reply:
I can revert that change. IntelliJ did the optimazation automatically because there were seven imports from the package